docs/advanced_topics/images/image_formats.rst

```diff
@@ -0,0 +1,49 @@
+Image formats
```

Comment:
I think this document could be merged into the main images docs. I wouldn't think this is an uncommon use case and we're documenting the settings there anyway

Reply:
Do you mean https://docs.wagtail.io/en/stable/topics/images.html ?

I could probably go there, but then that page is turning into a rather long one

Reply:
We should definitely rethink that document later, but I think it's the best place to document this feature in the short term

Reply:
👍

Reply:
I suggest we name this page "Image file formats" to differentiate it from the concept of image formats in rich text.